### PR TITLE
More small cross-browser visual fixes.

### PR DESCRIPTION
**Pull request by msecret**
- Extra line on page subsection on candidate page.
- More accurate alignment on accordion icon.
- Ensure there's always a px fallback for rem.

### CODE REVIEW

File: templates/partials/financial-summary.html [hunk 1]
```diff
@@ -6,7 +6,7 @@ <h2 id="section-1-header" tabindex="0">Financial Summary</h2>
       </div>
 
       {% if primary_committee or authorized_committees %}
-      <div class="page-subsection">
+      <div class="page-subsection page-subsection--noline">
```
Comment (noahmanger) [inline templates/partials/financial-summary.html]: I actually want the line here between the authorized committee section and the joint fundraising committees. Fine to keep the css class, but let's remove it from here.

File: static/styles/_base/_forms.scss [hunk 2]
```diff
@@ -49,13 +49,13 @@ select[multiple=multiple] {
   @include box-sizing(border-box);
   @include transition(border-color);
   @include font-size($form-font-size);
+  @include rem(padding, 1rem);
```
Comment (noahmanger) [inline static/styles/_base/_forms.scss]: Should actually be `padding: 0 1rem` but I fix this in a forthcoming pr.

File: templates/partials/financial-summary.html (hunk 1 above)
Comment (msecret) [inline templates/partials/financial-summary.html]: So this line is between primary and other committee:

screen shot 2015-05-04 at 2 33 58 pm

Is that what you meant? I wanted to check before I re-added it.

Comment (noahmanger) [inline templates/partials/financial-summary.html]: Yep!

**Pull request merged**